Adyen/Model/BalancePlatform/AccountHolder.cs

```diff
@@ -116,7 +116,7 @@ protected AccountHolder() { }
         /// Gets or Sets ContactDetails
         /// </summary>
         [DataMember(Name = "contactDetails", EmitDefaultValue = false)]
-        [Obsolete]
+        [Obsolete("Deprecated since Configuration API v - ")]
```

Comment:
@Kwok-he-Chu If there's no deprecated since maybe we should format this differently? Alternatively we can try to enforce this annotation on main if we think this is crucial enough

Reply:
Good catch. I have turned this into `[Obsolete("")]` for now (see [PR](https://github.com/Adyen/adyen-dotnet-api-library/pull/1068/files)), we'll enforce the specs to have the messages added from here on out (fix the problem at root-level)

Reply:
Fixed